src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -32,24 +33,47 @@ public MigrateCommand(string templateFile, string projectJson, string sdkVersion
 
         public int Execute()
         {
-            var project = GetProjectJsonPath(_projectJson) ?? GetProjectJsonPath(Directory.GetCurrentDirectory());
-            EnsureNotNull(project, "Unable to find project.json");
-            var projectDirectory = Path.GetDirectoryName(project);
+            var projectsToMigrate = GetProjectsToMigrate(_projectArg);
 
             var msBuildTemplate = _templateFile != null ?
                 ProjectRootElement.TryOpen(_templateFile) : _temporaryDotnetNewProject.MSBuildProject;
 
-            var outputDirectory = projectDirectory;
-
             var sdkVersion = _sdkVersion ?? new ProjectJsonParser(_temporaryDotnetNewProject.ProjectJson).SdkPackageVersion;
             EnsureNotNull(sdkVersion, "Null Sdk Version");
 
-            var migrationSettings = new MigrationSettings(projectDirectory, outputDirectory, sdkVersion, msBuildTemplate, _xprojFilePath);
-            new ProjectMigrator().Migrate(migrationSettings, _skipProjectReferences);
+            foreach (var project in projectsToMigrate)
+            {
+                Console.WriteLine($"Migrating project {project}..");
+                var projectDirectory = Path.GetDirectoryName(project);
+                var outputDirectory = projectDirectory;
+                var migrationSettings = new MigrationSettings(projectDirectory, outputDirectory, sdkVersion, msBuildTemplate, _xprojFilePath);
+                new ProjectMigrator().Migrate(migrationSettings, _skipProjectReferences);
+            }
 
             return 0;
         }
 
+        private IEnumerable<string> GetProjectsToMigrate(string projectArg)
+        {
+            if (projectArg.EndsWith(Project.FileName))
+            {
+                yield return GetProjectJsonPath(projectArg);
+            }
+            else if (Directory.Exists(projectArg))
+            {
+                var projects = Directory.EnumerateFiles(projectArg, Project.FileName, SearchOption.AllDirectories);
+
+                foreach(var project in projects)
+                {
+                    yield return GetProjectJsonPath(project);
+                }
+            }
+            else
+            {
+                throw new Exception($"Invalid project argument - {projectArg}");
```

Comment:
Can we make this error a little more informative?  Something like "'{projectArg}' is not a project.json file and a directory named '{projectArg}' doesn't exist".